From: Michael Tokarev <mjt@tls.msk.ru>
Subject: u-boot-sam460ex: fdi fix

Fix the missing extern in a variable declaration,
resulting in this variable being repeated each time
this header is included, so the link with modern gcc
fails.

diff --git a/roms/u-boot-sam460ex/board/ACube/common/vesa.h b/roms/u-boot-sam460ex/board/ACube/common/vesa.h
index a6c32c3c2c..99672aa67b 100644
--- a/roms/u-boot-sam460ex/board/ACube/common/vesa.h
+++ b/roms/u-boot-sam460ex/board/ACube/common/vesa.h
@@ -4,7 +4,7 @@
 extern void *DoVesa(int argc, char *argv[]);
 extern void *set_vesa_mode(int mode);
 
-struct FrameBufferInfo
+extern struct FrameBufferInfo
 {
         void *BaseAddress;
         unsigned long XSize;
